{"uid":"903e07ee851bce23","name":"Checking if the book information from the get request matches the books in the list","fullName":"tests.book_store_test.TestBookStore#test_comparing_book_information_from_get_request_and_book_list","historyId":"1245faf8bd6a3000c7f270fd25df5efe","time":{"start":1724581249244,"stop":1724581249244,"duration":0},"status":"failed","statusMessage":"AssertionError: 400","statusTrace":"@pytest.fixture(scope='function', autouse=True)\n    def account_setup():\n        account_api = AccountAPI()\n>       user_id = account_api.create_user()\n\ntests/conftest.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/account/api_account.py:25: in create_user\n    self.assert_response_status_code(response, 201)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.account.api_account.AccountAPI object at 0x7f6a0924e450>\nresponse = <Response [400]>, status_code = 201\n\n    def assert_response_status_code(self, response, status_code):\n        with allure.step(f\"Check response status code is {status_code}\"):\n>           assert response.status_code == status_code, response.status_code\nE           AssertionError: 400\n\nutils/helper.py:14: AssertionError","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1724581247058,"stop":1724581247127,"duration":69},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"account_setup","time":{"start":1724581249244,"stop":1724581249540,"duration":296},"status":"failed","statusMessage":"AssertionError: 400\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/usr/workspace/tests/conftest.py\", line 8, in account_setup\n    user_id = account_api.create_user()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/services/account/api_account.py\", line 25, in create_user\n    self.assert_response_status_code(response, 201)\n  File \"/usr/workspace/utils/helper.py\", line 14, in assert_response_status_code\n    assert response.status_code == status_code, response.status_code\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"Create user","time":{"start":1724581249244,"stop":1724581249540,"duration":296},"status":"failed","statusMessage":"AssertionError: 400\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/services/account/api_account.py\", line 25, in create_user\n    self.assert_response_status_code(response, 201)\n  File \"/usr/workspace/utils/helper.py\", line 14, in assert_response_status_code\n    assert response.status_code == status_code, response.status_code\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"Check response status code is 201","time":{"start":1724581249539,"stop":1724581249539,"duration":0},"status":"failed","statusMessage":"AssertionError: 400\n","statusTrace":"  File \"/usr/workspace/utils/helper.py\", line 14, in assert_response_status_code\n    assert response.status_code == status_code, response.status_code\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"feature","value":"Book Store"},{"name":"epic","value":"API Tests"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"book_store_test"},{"name":"subSuite","value":"TestBookStore"},{"name":"host","value":"af9534f9e50a"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.book_store_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":2},"items":[{"uid":"8b3a143fc24b339","reportUrl":"https://vvkobzar.github.io/aqa_pet_project/32//#testresult/8b3a143fc24b339","status":"passed","time":{"start":1724514744330,"stop":1724514744930,"duration":600}}]},"tags":[]},"source":"903e07ee851bce23.json","parameterValues":[]}